src/models/payments.js

```diff
@@ -316,13 +313,13 @@ function * settle (payment) {
   }
 }
 
-function isTraderFunds (funds) {
+function isTraderFunds (config, funds) {
```

Comment:
Because later it's used as a callback function, so I was able to do `_.partial(isTraderFunds, config)`